charmcraft.yaml

```diff
@@ -49,22 +49,53 @@ requires:
 
 config:
   options:
+    cni-type:
+      type: string
+      default: bridge
+      description: |
+        Multus CNI plugin to use for the interfaces.
+        Allowed values are `bridge`, `macvlan`.
     f1-interface-name:
       type: string
-      default: "eth0"
-      description: Name of the network interface handling communication over the F1 interface.
+      default: "f1"
+      description: |
+        Name of the network interface handling communication over the F1 interface.
+        Host interface to use for the DU Network. If not specified, a bridge will be used.
+    f1-ip-address:
+      type: string
+      default: "192.168.251.7/24"
+      description: CU F1 interface IP Address
     f1-port:
       type: int
       default: 2153
       description: Number of the port handling communication over the F1 interface.
     n2-interface-name:
       type: string
-      default: "eth0"
-      description: Name of the network interface handling communication over the N2 interface.
+      default: "n2"
+      description: |
+        Name of the network interface handling communication over the N2 interface.
+        Host interface to use for the Core Network. If not specified, a bridge will be used.
+    n2-ip-address:
+      type: string
+      default: "192.168.250.6/24"
+      description: CU N2 interface IP Address in CIDR format
     n3-interface-name:
       type: string
-      default: "eth0"
-      description: Name of the network interface handling communication over the N3 interface."
+      default: "n3"
+      description: | 
+        Name of the network interface handling communication over the N3 interface. 
+        Host interface to use for the RAN Network. If not specified, a bridge will be used.
+    n3-ip-address:
+      type: string
+      default: "192.168.251.6/24"
+      description: CU N3 interface IP Address in CIDR format
+    n3-gateway-ip:
+      type: string
+      description: |
+        Gateway IP to the UPF Access Network. This should be provided, if a route is required to UPF access network subnet.
+    n3-subnet:
```

Comment:
Not sure about the name here.
It will be confusing, because the N3 IP address will likely not belong to this subnet. How about something like `upf-subnet`?

Reply:
We can rename it  as upf-subnet. This is fine.

Reply:
Done